common/src/main/java/org/apache/celeborn/common/network/server/BaseMessageHandler.java

```diff
@@ -27,6 +28,10 @@ public void receive(TransportClient client, RequestMessage msg) {
     throw new UnsupportedOperationException();
   }
 
+  public void receive(TransportClient client, RequestMessage msg, RpcResponseCallback callback) {
```

Comment:
Currently, `callback` is getting ignored - is this expected to be leveraged in future pr's ?
If yes, add a comment until then ?

Reply:
It is not getting ignored. I have made changes to the extensions - `NettyRpcHandler` to use it. We will use it in `SaslRpcHandler` as well. Draft PR: https://github.com/apache/incubator-celeborn/pull/2105

Reply:
Sounds good - so we will use it later as a fast follow.

Reply:
Yes, we will use it.